src/renderer/commands/createDrawGuidingCurve.ts

```diff
@@ -54,21 +68,38 @@ export function createDrawGuidingCurve(
             }
         `,
         attributes: {
-            position: regl.prop<DrawGuidingCurveProps, keyof DrawGuidingCurveProps>('positions')
+            position: (_context: REGL.DefaultContext, props: DrawGuidingCurveProps) =>
+                flatMap(props.positions, (position: number) => [position, position]),
+            side: (_context: REGL.DefaultContext, props: DrawGuidingCurveProps) =>
+                flatMap(props.positions, (_position: number) => [-1, 1]),
+            direction: (_context: REGL.DefaultContext, props: DrawGuidingCurveProps) =>
+                range(props.positions.length * 2).map((i: number) => {
```

Comment:
nit: I think it would be a bit cleaner to do a flatMap here, but your call

Reply:
sure, itll be consistent with the others